File: 백준/Bronze/10810. 공 넣기/공 넣기.py
ball_num, count = map(int, input().split())
nums = [0] * ball_num   # 0으로 채워진 리스트

for _ in range(count):
    i, j, k = map(int, input().split())
    for index in range(i-1, j):
        nums[index] = k

print(" ".join(map(str, nums)))

File: 백준/Bronze/10810. 공 넣기/공 넣기.js
const input = require("fs")
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split("\n");

const [N, M] = input[0].split(" ").map(Number);
const arr = new Array(N).fill(0);

for (let m = 1; m <= M; m++) {
  const [i, j, k] = input[m].split(" ").map(Number);

  for (let x = i - 1; x <= j - 1; x++) {
    arr[x] = k;
  }
}

console.log(arr.join(" "));
